### PR TITLE
CP-2821 Documentation tweaks

### PR DESCRIPTION
**Pull request by evanweible-wf**
Just a few more tweaks to the documentation prior to 3.0.0.

**Docs change only.**

@trentgrover-wf 
@maxwellpeterson-wf 
@dustinlessard-wf 
@jayudey-wf 
@sebastianmalysa-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
Number of Findings: 0

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ dependencies:
   w_transport: ^3.0.0
 ```
 
-If you're upgrading from version 2.x, you should be able to use the above
-version range without breaking any existing code. **Check out the [3.0.0
-upgrade guide](/docs/upgrade-guides/v3.0.0.md)**.
+If you're upgrading from version 2.x, you can use the above version range
```
Comment (evanweible-wf) [inline README.md]: @dustinlessard-wf you had requested the stronger verbiage here and I forgot to make that change prior to merging the original docs PR.

**Comment by maxwellpeterson-wf** [comment]
+1

File: docs/upgrade-guides/v3.0.0.md [hunk 2]
```diff
@@ -130,3 +130,16 @@ being removed in version 4.
   available. They provided a way to control mock instances of requests and
   WebSockets, but it required having access to the instances (which isn't always
   possible) and required type checks/casts. They will be removed in version 4.
+  
+  If you were constructing any of these classes, the correct way to update your
+  usage is to first install the mock transports, and then construct the
+  corresponding class from the main `w_transport.dart` entry point.
+  
+  ```diff
+  + MockTransports.install();
```
Comment (trentgrover-wf) [inline docs/upgrade-guides/v3.0.0.md]: would read a tiny bit better with some strategic whitespace:
```
  + MockTransports.install();

  - new MockClient();
  + new transport.HttpClient();

  - new MockJsonRequest();
  + new transport.JsonRequest();
```

Comment (evanweible-wf) [inline docs/upgrade-guides/v3.0.0.md]: Agreed, and fixed

**Comment by trentgrover-wf** [comment]
+1
QA +10
docs only change so merging...

**Comment by evanweible-wf** [comment]
run_merge_script

**Pull request merged**